PeerioTechnologies/peerio-mobile
# fix: changed header main icon and refactor layout [branch ch5788]

#### Relevant info and issue/PR links 
https://app.clubhouse.io/peerio/story/5788/mobile-update-dropdown-arrow-no-dm-and-room-bar
#### Testing instructions  
Make sure main header buttons and title press action still works in the following places:
chats (dm and room and "+" button)

----
### Repository owner


[PR guideline](https://github.com/PeerioTechnologies/peerio-icebear/blob/dev/docs/CONTRIBUTING.md)
